bundles/binding/org.openhab.binding.xbmc/src/main/java/org/openhab/binding/xbmc/rpc/XbmcConnector.java

```diff
@@ -276,15 +278,16 @@ public void run() {
 
     /**
      * Create a mapping between an item and an xbmc property
-     * 
+     *
      * @param itemName
      *            The name of the item which should receive updates
      * @param property
      *            The property of this xbmc instance, which is to be
      *            watched for changes
-     * 
+     *
      */
     public void addItem(String itemName, String property) {
+		logger.debug("Mapping: itemname = {} & property = {}", itemName, property);
```

Comment:
Please put the code through the Eclipse IDE formatter so future changes don't reformat lines with incorrect formatting.